fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -102,15 +102,15 @@ private PostLoanProductsRequest() {}
         @Schema(example = "false")
         public Boolean useBorrowerCycle;
         @Schema(example = "2")
-        public Integer repaymentFrequencyType;
+        public Long repaymentFrequencyType;
```

Comment:
This looks wrong to me based on it is the ordinal of an enum.

Reply:
Done.